src/main/java/org/sagebionetworks/bridge/sdk/LowercaseEnumDeserializer.java

```diff
@@ -0,0 +1,29 @@
+package org.sagebionetworks.bridge.sdk;
+
+import java.io.IOException;
+import java.lang.reflect.Method;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.deser.std.StdScalarDeserializer;
+
+@SuppressWarnings("serial")
+class LowercaseEnumDeserializer extends StdScalarDeserializer<Enum<?>> {
+
+    protected LowercaseEnumDeserializer(Class<Enum<?>> clazz) {
+        super(clazz);
+    }
+
+    @Override
+    public Enum<?> deserialize(JsonParser jp, DeserializationContext ctxt) throws IOException, JsonProcessingException {
+        String text = jp.getText().toUpperCase();
+        try {
+            Method valueOfMethod = handledType().getDeclaredMethod("valueOf", String.class);
+            return (Enum<?>) valueOfMethod.invoke(null, text);
+        } catch (Exception e) {
+            throw new RuntimeException("Cannot deserialize enum " + handledType().getName() + " from " + text, e);
+        }
```

Comment:
What's the compilation error you're getting for Enum.valueOf()? If I were to guess, I'd guess you'd need to cast handledType() to a Class<? extends Enum<?>> or something similar.

I'm confused by "adds the endpoint that failed to the exception". As far as I can tell, IllegalArgumentException or JsonProcessingException both accurately and specifically label this exception.